webpack.config.babel.js

```diff
@@ -167,22 +158,20 @@ const createConfig = (env = {}) => {
           exclude: /node_modules/,
           loader: 'babel-loader'
         },
-        {
-          test: inCurrentRepo('css'),
-          exclude: /node_modules/,
-          use: [
-            'isomorphic-style-loader',
-            cssModuleLoader,
-            'postcss-loader'
-          ],
-        },
         {
           test: inCurrentRepo('scss'),
           exclude: /node_modules/,
           use: [
             'isomorphic-style-loader',
-            cssModuleLoader,
-            'postcss-loader',
```

Comment:
Jeg tror vi trenger postcss og autoprefixer for scss, altså...

Reply:
Ja kanskje en litt forhastet beslutning der... Jeg skal også gå tilbake på node-sass også pga en warning som det er store diskusjoner om på deres github-side